src/frontend/components/Felleskomponenter/Barnehagemodal/BarnehageplassPeriode.tsx

```diff
@@ -32,40 +35,47 @@ export const BarnehageplassPeriode: React.FC<BarnehageplassPeriodeProps> = ({
 }) => {
     const { erÅpen: barnehageplassModalErÅpen, toggleModal: toggleBarnehageplassModal } =
         useModal();
+    const { tekster } = useApp();
+    const barnehageplassTekster: IBarnehageplassTekstinnhold =
+        tekster()[ESanitySteg.FELLES].modaler.barnehageplass;
+    const oppfølgingstekst: IOmBarnetTekstinnhold = tekster()[ESanitySteg.OM_BARNET];
```

Comment:
Denne variabelen er mer enn oppfølgningsteksten nå, den er alle tekstene for OM_BARNET. Så ville enten kalt den noe annet, eller hentet ut den faktisk oppfølgningsteksten her 😊